### ISSUE TITLE
registration 

### ISSUE THREAD

**Issue by slmyers** [401ChemistryGenealogy/ChemistryGenealogy#2]
Should we implement email validation in the registration process?

**Comment by theresa-cy-ngo** [comment]
I definitely like the idea, but only if we have time?

**Comment by jihwan1ua** [comment]
We could, but this should be done on later spints.
I guess for now we can just make a test cases for validation.

**Comment by slmyers** [comment]
We're going to need to implement some type of notification for the user of when they're approved by the admin. This is basically now a requirement.

**Comment by theresa-cy-ngo** [comment]
Agreed. Do you think it'll be easier to make it an email notification or should we work off of what we'll have for admin notifications?

**Comment by slmyers** [comment]
email notification is the only real option

**Comment by theresa-cy-ngo** [comment]
Okay, email notification it is then.
